File: Python/Kode Program/Jus.py
from Items import Items

class Jus(Items):
    def __init__(self, Id="", Nama="", Kategori="", HargaProduk=0, Buah=""):
        super().__init__(Id, Nama, Kategori, HargaProduk)
        self.__Buah = Buah

    def set_buah(self, buah1):
        self.__Buah = buah1

    def get_buah(self):
        return self.__Buah

File: CPP/Kode Program/Jus.cpp
#ifndef JUS_CPP
#define JUS_CPP

#include <iostream>
#include <string>
#include "Items.cpp"

using namespace std;

class Jus : public Items
{
    private:
        string Buah;      // Jenis buah yang digunakan (contoh: "Mangga", "Jeruk", "Apel")

    public:
        // Constructor Default
        Jus() 
        {

        }

        // Setter dan Getter Buah
        void setBuah(string buah1)
        {
            Buah = buah1;
        }

        string getBuah()
        {
            return Buah;
        }

        ~Jus()
        {

        }
};

#endif  // JUS_CPP